### ISSUE TITLE
Separate i18n from Trails core

### ISSUE THREAD

**Issue by tjwebb** [trailsjs/trails#274]
#### Issue Description
Originally, I thought that i18n (internationalization) support should be a core feature of trails. Due to the complexity and diversity of options for i18n, I think it should be moved to a trailpack

**Comment by jaumard** [comment]
Yeah I don't see why i18n should be a core feature, so let's put it on a trailpack :) that will allow us to add more i18n functionalities without pollute Trails's code

**Comment by weyert** [comment]
Sounds like a good idea as it's cause a vulnerability in trails:

Cross-site Scripting (XSS)

Medium severity
Vulnerable module: i18next
Introduced through: i18next@3.4.1
Detailed paths

Introduced through: trails@2.0.1 › i18next@3.4.1
Overview

i18next is i18next internationalization framework. Affected versions of the package are vulnerable to Cross-site Scripting (XSS) due to not escaping variables as expected. When passing any interpolation options without including escapeValue, the function this.escapeValue will get set to undefined in Interpolator. Meaning values aren't being escaped even though users expect them to be.

**Comment by jaumard** [comment]
Maybe this can be planned for v3 ? Don't know if you already remove it @tjwebb
